frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,164 @@
+import SearchIcon from '@mui/icons-material/Search'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+import Grid from '@mui/material/Grid'
+import IconButton from '@mui/material/IconButton'
+import InputBase from '@mui/material/InputBase'
+import MuiLink from '@mui/material/Link'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import { useTheme } from '@mui/material/styles'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import Typography from '@mui/material/Typography'
+import Link from 'next/link'
+import React, { useState } from 'react'
+import TagSelector from 'src/common/TagSelector'
+
+import { ListModelType, useListModels } from '../../data/model'
+import EmptyBlob from '../../src/common/EmptyBlob'
+import MultipleErrorWrapper from '../../src/errors/MultipleErrorWrapper'
+import Wrapper from '../../src/Wrapper'
+import { Model, Version } from '../../types/types'
+import useDebounce from '../../utils/hooks/useDebounce'
+
+export default function ExploreModels() {
+  const [group, setGroup] = useState<ListModelType>('all')
+  const [filter, setFilter] = useState('')
+  const debouncedFilter = useDebounce(filter, 250)
+
+  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+
+  const theme = useTheme()
+
+  const error = MultipleErrorWrapper(`Unable to load marketplace page`, {
+    isModelsError,
+  })
+  if (error) return error
+
+  const handleFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFilter(e.target.value)
+  }
+
+  const onFilterSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+  }
+
+  const updateSelectedTags = (selectedTags: string[]) => {
+    console.log(selectedTags)
+  }
+
+  const updateSelectedType = (selectedType: string) => {
+    if (selectedType) {
+      switch (selectedType) {
+        case 'My models':
```

Comment:
Would be better to use enums rather than "magic" strings. That way we only have to update the enum if we change these in future and those changes will propagate across the app.